TMessagesProj/src/main/res/values/cg_strings.xml

```diff
@@ -322,9 +321,9 @@
     <string name="CG_MentionByName">Mention by name</string>
     <string name="CG_MentionByName_Desc">instead of username</string>
     <string name="CG_PlayVideo">Play video on volume button click</string>
-    <string name="CG_PlayVideo_Desc">If turned on - when you press the volume button, the video on the screen in the dialog will be enabled</string>
+    <string name="CG_PlayVideo_Desc">Use volume buttons to start video playback when in chat.</string>
     <string name="CG_StreamVoip">Stream VoIP as Media</string>
-    <string name="CG_StreamVoip_Desc">if enabled then VoIP call will be recognized as media instead of call</string>
+    <string name="CG_StreamVoip_Desc">Turn this on to get a boost in voice quality when using Bluetooth to take calls.</string>
```

Comment:
Little unclear where Bluetooth came from here

Reply:
I think I tested this one to come to that conclusion.